pkg/scoring/atlassian.go

```
@@ -0,0 +1,339 @@
+	req, err := http.NewRequestWithContext(ctx, "GET", url, nil)
+	if err != nil {
+		return false, nil
+	}
+	req.SetBasicAuth(email, token)
+
+	client := c.httpClient()
+	resp, err := client.Do(req)
+	if err != nil {
+		return false, nil
+	}
+	defer func() { io.Copy(io.Discard, resp.Body); resp.Body.Close() }()
```

golangci-lint (2.12.2)

[error] Error return value of `io.Copy` is not checked

(errcheck)